packages/trader/src/AppV2/Components/Guide/definition-modal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ActionSheet, Heading, Text } from '@deriv-com/quill-ui';
+import { Localize, localize } from '@deriv/translations';
+import { getTermDefinition } from 'AppV2/Utils/trade-types-utils';
+
+type TDefinition = { contract_type: string; term: string; onClose: () => void };
+
+const DefinitionModal = ({ contract_type, term, onClose }: TDefinition) => (
+    <ActionSheet.Root isOpen={!!term} onClose={onClose} position='left'>
+        <ActionSheet.Portal shouldCloseOnDrag>
+            <ActionSheet.Content className='definition__wrapper'>
+                <Heading.H4 className='definition__title'>{localize(term)}</Heading.H4>
```

Comment:
it's not gonna localize the term variable. the terms should be localized in the config where they are declared.

Reply:
Yep, I'm wrong, will replace wrapped with localize()